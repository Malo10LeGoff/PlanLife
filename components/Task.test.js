import React from 'react';
import renderer from 'react-test-renderer';
import Task from './Task';

test('renders correctly', () => {
    const tree = renderer.create(<Task text={"mock_task"} />).toJSON();
    expect(tree).toMatchSnapshot();
});