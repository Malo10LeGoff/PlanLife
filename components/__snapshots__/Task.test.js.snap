// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<View
  style={
    Object {
      "alignItems": "center",
      "backgroundColor": "#FFF",
      "borderRadius": 10,
      "flexDirection": "row",
      "justifyContent": "space-between",
      "marginBottom": 20,
      "padding": 15,
    }
  }
>
  <View
    style={
      Object {
        "alignItems": "center",
        "flexDirection": "row",
        "flexWrap": "wrap",
      }
    }
  >
    <View
      style={
        Object {
          "backgroundColor": "#55BCF6",
          "borderRadius": 5,
          "height": 24,
          "marginRight": 15,
          "opacity": 0.4,
          "width": 24,
        }
      }
    />
    <Text
      style={
        Object {
          "maxWidth": "80%",
        }
      }
    >
      mock_task
    </Text>
  </View>
  <View
    style={
      Object {
        "borderColor": "#55BCF6",
        "borderRadius": 5,
        "borderWidth": 2,
        "height": 12,
        "width": 12,
      }
    }
  />
</View>
`;
